contracts/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@t1protocol/t1-contracts",
   "description": "Solidity code for T1's L1 bridge and rollup contracts, plus L2 bridge and pre-deployed contracts.",
-  "version": "0.1.0",
+  "version": "0.1.1",
```

Comment:
I would wait to do this when releasing. Perhaps, we should review the the pull request template.

Reply:
[this comment from Orest](https://github.com/t1protocol/t1/pull/69#issuecomment-2737200041) indicates he would prefer that we bump each time, which I also like.